frontend/packages/dev-console/src/components/project-access/ProjectAccess.tsx

```diff
@@ -108,8 +114,9 @@ const ProjectAccess: React.FC<ProjectAccessProps> = ({ formName, namespace, role
     <>
       <PageHeading>
         <Trans t={t} ns="devconsole">
-          Project access allows you to add or remove a user&apos;s access to the project. More
-          advanced management of role-based access control appear in{' '}
+          {
+            "Project access allows you to add or remove a user's access to the project. More advanced management of role-based access control appear in "
+          }
```

Comment:
Is this change requested by someone or can we keep the old version?

Also if we don't have a translation for this strings now, I would personally avoid unnecessary changes to translation.

If there is a reason for that I'm absolutely fine with that change.

Reply:
This change fixes the `missing i18n key` error on the project access page.